packages/plugins/content/html5-video/src/Component/index.js

```diff
@@ -51,10 +51,11 @@ const HTML5Video = ({
           label="Video url"
           onChange={changeUrl(onChange)}
           value={url}
+          style={{ width: '512px' }}
```

Comment:
Maybe have that as minWidth? It would be even better if that would be a responsive size as we could view the video on even smaller screens

Reply:
Thanks for the feedback!
I did think about using relative widths (e.g. 80%), somewhat consistent with ory's cells. But in the end, the scope of this PR was just fixing urgent matters, so I reused the exact style of the other plugins' form (`video` and `image`).
Imo, decisions to change the forms' behaviour should be taken to a different PR which would alter all plugins consistently

Reply:
Hm, but the video image uses relative width: https://github.com/aeneasr/ory-editor/blob/master/packages/plugins/content/video/src/Component/Display/index.js#L35-L52

Reply:
Same for the [image plugin](https://github.com/aeneasr/ory-editor/blob/master/packages/plugins/content/image/src/Component/Display/index.js#L33-L49) as well as [here](https://github.com/aeneasr/ory-editor/blob/master/packages/plugins/content/image/src/Component/index.css#L23-L31)

Reply:
But that's a different thing: this width is the form's, not the player's.
Here's the relevant code for the [image plugin](https://github.com/aeneasr/ory-editor/blob/master/packages/plugins/content/image/src/Component/Form/index.js#L114) and the [video plugin](https://github.com/aeneasr/ory-editor/blob/master/packages/plugins/content/video/src/Component/Form/index.js#L47)

Reply:
omg yeah, I completely overlooked that. Sorry! :)

Reply:
yeah, I got that, no worries :)